### PR TITLE
Dirft v2 part 2

### PR DESCRIPTION
**Pull request by tinystarinagalaxy**

### CODE REVIEW

File: bxsolana/examples/request_utils.py [hunk 1]
```diff
@@ -316,7 +316,161 @@ async def do_requests(
...
+                        "9UnwdvTf5EfGeLyLrF4GZDUs7LKRUeJQzW7qsDVGQ8sS"
+                    ),
+                    contract="SOL_PERP",
+                    amount=1,
+                )
+            )
+        ).to_json()
+    )
+
+    print("get Drift perp orderbook")
```
Comment (DimaCantemir) [inline bxsolana/examples/request_utils.py:406]: Drift -> drift  ? Not sure why you made all "drift" lower case but I suppose you missed this one

File: bxsolana/examples/request_utils.py [hunk 2]
```diff
@@ -316,7 +316,161 @@ async def do_requests(
...
+        (
+            await api.get_drift_open_perp_order(
+                get_drift_open_perp_order_request=proto.GetDriftOpenPerpOrderRequest(
+                    owner_address=public_key, client_order_id=12, order_id=1
+                )
+            )
+        ).to_json()
+    )
+
+    print("post Drift get open margin order")
```
Comment (DimaCantemir) [inline bxsolana/examples/request_utils.py:459]: OK, one more commend and I'll stop - make these Drift vs drift consistent please

Comment (tinystarinagalaxy) [reply bxsolana/examples/request_utils.py:459]: done

File: bxsolana/examples/request_utils.py (hunk 1 above)
Comment (tinystarinagalaxy) [reply bxsolana/examples/request_utils.py:406]: done, all capitalized now

**Review by DimaCantemir — APPROVED** [review]

**Pull request merged**